plugwise/helper.py

```diff
@@ -46,20 +56,8 @@
     version_to_model,
 )
 
-_LOGGER = logging.getLogger(__name__)
-
-DAYS = {
-    "mo": 0,
-    "tu": 1,
-    "we": 2,
-    "th": 3,
-    "fr": 4,
-    "sa": 5,
-    "su": 6,
-}
-
 
-def pw_notification_updater(devs, d_id, d_dict, notifs):
+def pw_notification_updater(devs, d_id, d_dict, notifs) -> None:
```

Comment:
Arguments are missing typing (same for other methods)

Reply:
That's a helper function, the arguments are typed in the main function.
The same is valid for all other helper functions.

Reply:
`_request()`, `_presets()`, and `_schemas()` are the only functions of which the arguments need to be typed I think.
Now that I think about it, these arguments should also "receive" their typing in the calling functions in `smile.py`.

Reply:
arguments do not inherit typing. They must be typed.

Reply:
Indeed they do, relevant typing hints were removed.